UPDATING.md

```diff
@@ -24,6 +24,15 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Merge subpackages of cloud providers
+
+Subpackages for specific services have been combined into one variant for the entire cloud.
+If you want to install integration for Microsoft, then instead of `pip install airflow[azure_blob_storage,azure_data_lake,azure_cosmos,azure_container_instances]`, 
```

Comment:
```suggestion
If you want to install integration for Microsoft Azure, then instead of `pip install airflow[azure_blob_storage,azure_data_lake,azure_cosmos,azure_container_instances]`, 
```

Reply:
Line is a bit long too. Please wrap (probably before the code block, after "of")